l10n_ch_payroll_attendances/README.rst

```diff
@@ -0,0 +1,44 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Attendances and timesheet improvements
+======================================
+
+This module allows you to add commission and reimbursement to payslip
+
+**Features list :**
+    * Add an hourly rate as a base for salary compensation computing
+    * Add a link payslip to hr_holiday
+    * Compute payslip attendances based on employe's schedule and employe's attendance
+
+Usage
+=====
+- To start using this module, you need to configure a contract on your employees.
+- This contract must contain a Working Schedule.
+
+**For exemple :**
+    .. image:: http://s23.postimg.org/hkzqbz83f/Screen_Shot_04_13_16_at_02_38_PM.png
+
+- Here you can see in red the fields added by the module.
+- These fields are used in employee's payslip to determin time and wage compensation.
+- Once you've configured the working schedule for each employee.
+- You can create payslip for your employees.
+
+**For exemple :**
+    .. image:: http://s23.postimg.org/cghcwp5mj/Screen_Shot_04_13_16_at_02_58_PM.png
+
+- Here you can see in red two new elements : 
+    - A new button to compute the attendances
+    - A new tab to see the result of the computation
+- Into the result of the computation tab you can see something like this :
+
+.. image:: http://s28.postimg.org/bittrg17h/Screen_Shot_04_13_16_at_03_05_PM.png
+
+- This result is used to create allocation request if time compensation was set.
+- It is also used to add salary compensation if you correctly set the corresponding rule into the salary structur of the employee.
+
+
+Contributors
+------------
+
+* David Coninckx <dco@open-net.ch>

```

Comment:
Please see the template, there are some missing informations:

https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst